.travis.yml

```diff
@@ -52,7 +52,6 @@ matrix:
     - {python: 3.6, env: TOXENV=py36-django19-nomigrations}
     - {python: 2.7, env: TOXENV=py27-django111-nomigrations}
     - {python: 3.5, env: TOXENV=py35-django111-nomigrations}
-    - {python: 3.5, env: TOXENV=py35-django111-nomigrations}
```

Comment:
Likely this should have been py 3.6 without migrations since seems that we have one for Django 1.9 too.